### PR TITLE
[FEATURE-UIPT-67]: PLP - two views for products

### PR DESCRIPTION
**Pull request by castan1c**

### CODE REVIEW

File: src/app/shared/product-filter/product-filter.html [hunk 1]
```diff
@@ -1,47 +1,45 @@
...
-                  <fa-icon class="icon" [icon]="checkboxIcon">
-                  </fa-icon>
-                </span>
+    <h2 class="filter-criteria__title">Brand</h2>
+    <div class="filter-criteria__wrapper">
+      <div class="filter-criteria" *ngFor="let product of productBrand">
+        <div class="center">
+          <label class="label">
+            <input class="filter-criteria__option label__checkbox"
+              (change)="onCreateRequest($event)"
```
Comment (andriiveremchuk) [inline src/app/shared/product-filter/product-filter.html]: Use ordering:
```
*structuralDirective
origin-attribute
[binding]
(method)
```
Check everywhere

File: src/app/shared/product-list/product-list.html [hunk 2]
```diff
@@ -1,31 +1,9 @@
...
-        class="product__item"
-        [status]="product.status"
-        [imgUrl]="product.imgUrl"
-        [imgUrlNext]="product.imgUrlNext"
-        [productTitle]="product.productTitle"
-        [productPrice]="product.productPrice"
-        [productId]="product.productId">
-      </app-product-item>
-    </div>
+    <h2 class="col-12 content__title">{{ listTitle }}</h2>
```
Comment (andriiveremchuk) [inline src/app/shared/product-list/product-list.html]: Can you align like that ?
```
<h2 class="col-12 content__title">
  {{ listTitle }}
</h2>
```

File: src/app/shared/shop-by-category/shop-by-category.html [hunk 3]
```diff
@@ -1,25 +1,11 @@
...
-        [imgUrl]="product.imgUrl"
-        [imgUrlNext]="product.imgUrlNext"
-        [productTitle]="product.productTitle"
-        [productPrice]="product.productPrice"
-        [productId]="product.productId"
-        class="product__item">
-      </app-product-item>
-    </div>
+  <div class="products__wrapper">
+    <h2 class="content__title">{{ filterCategory | titlecase }} Collection</h2>
```
Comment (andriiveremchuk) [inline src/app/shared/shop-by-category/shop-by-category.html]: Same here, but it was actually my fault :)

File: src/app/shared/product-list/product-list.component.ts [hunk 4]
```diff
@@ -1,4 +1,4 @@
-import { Component, OnInit, OnDestroy, HostListener } from '@angular/core';
+import { Component, OnInit, OnDestroy, HostListener, Input } from '@angular/core';
```
Comment (TorskyM) [inline src/app/shared/product-list/product-list.component.ts]: Align multiple imports

File: src/app/shared/wish-button/wish-button.scss [hunk 5]
```diff
@@ -16,6 +16,15 @@
       color: $fontColorDark;
       opacity: 0.5;
```
Comment (TorskyM) [inline src/app/shared/wish-button/wish-button.scss]: Style ordering

File: src/app/shared/shop-by-category/shop-by-category.scss [hunk 6]
```diff
@@ -1,84 +1,36 @@
...
-  .product {
-    &__container {
-      margin: 0;
-      }
-
-    &__navigation {
-      height: 200px;
-    }
-  }
+@media screen and (max-width: 880px) {
```
Comment (TorskyM) [inline src/app/shared/shop-by-category/shop-by-category.scss]: Empty media query

File: src/app/shared/products-view/products-view.scss [hunk 7]
```diff
@@ -0,0 +1,37 @@
...
+}
+
+
+@media screen and (max-width: 992px) {
+  .button-four {
+    display: none;
+  }
+}
+
+@media screen and (max-width: 425px) {
```
Comment (IrynaMukhina) [inline src/app/shared/products-view/products-view.scss]: please use max-width: 576px

**Pull request merged**